src/key_schedule.rs

```diff
@@ -232,25 +233,7 @@ pub(crate) fn derive_traffic_iv(secret: &hkdf::Prk) -> Iv {
 mod test {
     use super::{KeySchedule, SecretKind, derive_traffic_key, derive_traffic_iv};
     use ring::{aead, hkdf};
-
-    #[test]
-    fn smoke_test() {
```

Comment:
I removed this test because I don't see its value given the other test. It was testing a function that's been removed.